Doc/library/shutil.rst

```diff
@@ -559,6 +559,10 @@ provided.  They rely on the :mod:`zipfile` and :mod:`tarfile` modules.
 
    The *verbose* argument is unused and deprecated.
 
+   .. versionchanged:: 3.8
+      The modern modern pax (POSIX.1-2001) format is now used instead of
```

Comment:
Was doubling _modern_ meant to distract me from noticing 2001 is eighteen years ago? :)

Reply:
Heh. "modern"-est, I suppose, although to be fair it took even libarchive (modern ``bsdtar``) 10 years to implement fully complete support (ironically, GNU tar had it far earlier). And would you believe I've worked as a freelance copyeditor?

In any case, fixed, thanks.